docs/source/plotly_figures/plot_param_importances.html

```diff
@@ -0,0 +1,20 @@
+<div>
+        
+                <script type="text/javascript">window.PlotlyConfig = {MathJaxConfig: 'local'};</script>
```

Comment:
Oh you are right. Might be an issue on plotly since this file was generated automatically by it. It's also consistent across other generated files https://github.com/optuna/optuna/tree/master/docs/source/plotly_figures. Do you still want me to modify it?

Reply:
> Might be an issue on plotly since this file was generated automatically by it.

Thank you for your explanation. I think we can leave as it is.